elide-datastore/elide-datastore-hibernate3/src/main/java/com/yahoo/elide/datastores/hibernate3/HibernateTransaction.java

```diff
@@ -466,7 +466,14 @@ private Query populateWhereClause(String hqlQuery, FilterExpression expression)
         for (FilterPredicate filterPredicate : predicates) {
             if (filterPredicate.getOperator().isParameterized()) {
                 String name = filterPredicate.getParameterName();
-                query = query.setParameterList(name, filterPredicate.getValues());
+                if(filterPredicate.getOperator() == Operator.INFIX || filterPredicate.getOperator() == Operator.INFIX_CASE_INSENSITIVE ||
+                        filterPredicate.getOperator() == Operator.PREFIX || filterPredicate.getOperator() == Operator.PREFIX_CASE_INSENSITIVE ||
+                        filterPredicate.getOperator() == Operator.POSTFIX || filterPredicate.getOperator() == Operator.POSTFIX_CASE_INSENSITIVE){
```

Comment:
Let's add a method like `Operator.isStringMatching` that encapsulates this logic.